docs/howto_guides/globus.md

```diff
@@ -0,0 +1,19 @@
+# Downloading NMDC Data via Globus
+
+[Globus](https://globus.org) provides a mechanism to download NMDC data using high-bandwidth managed transfers. Globus has an automated point-and-click interface that lets you schedule a bulk transfer to your own machine or another compute center. You can learn more about using Globus to transfer data, by reading the [Globus documentation](https://www.globus.org/data-transfer).
+
+## Globus Collections
+
+NMDC collections are publicly visible to everyone with a Globus ID. You can create a Globus ID at https://www.globusid.org/create.
+
+### NERSC
+
+To access NMDC data housed at NERSC, please visit the ["NMDC" collection](https://app.globus.org/file-manager?origin_id=72dd396a-2242-11ec-a0a4-6b21ca6daf73&origin_path=%2F).
+
+In this collection, NMDC data are organized by _NMDC identifiers_. This collection serves up the same contents as https://data.microbiomedata.org/data/ does, so any file path underneath that base URL can be mapped to the equivalent file in Globus.
```

Comment:
I think including one example of an NMDC identifier here could be useful to some readers.